docs/configuration.md

```diff
@@ -67,26 +67,46 @@ The following options can be configured via environment variables.
+The configuration file is a _single yaml_ file containing all the read sides settings one desires. The following settings are required
+to define a read side configuration file:
+
+| Setting    | Required                      | Description                                                                                                                                                                                                                                               |
+|------------|-------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| readSideId | yes                           | Specifies the read side unique identifier. The read side must only contain alphanumeric characters plus hyphens and underscores. No spacing is allowed.                                                                                                   |
+| host       | yes                           | Specifies the read side host address                                                                                                                                                                                                                      |
+| port       | yes                           | Specifies the read side port number                                                                                                                                                                                                                       |
+| useTls     | no (default value is `false`) | Specifies whether to use TLS to connect to the read side                                                                                                                                                                                                  |
+| autoStart  | no (default value is `true`)  | Set to `true` means that the Read side on start is ready to process events. However, when it set to `false` means that the Read side is paused on start or no not. One can use the [cli](https://github.com/chief-of-state/cos-cli) to resume processing. |
+
+#### Example: read-side-config.yml
+```yaml
+readSideId: read-side-1
+host: read-handler
+port: 50053
+useTls: false
+autoStart: true
+---
+readSideId: read-side-2
+host: read-handler
+port: 50054
+useTls: false
+autoStart: true
+```
 
-| environment variable                                | description                     | default |
-|-----------------------------------------------------|---------------------------------|---------|
-| COS_READ_SIDE_CONFIG__<SETTING_NAME>__<READSIDE_ID> | readside configuration settings | <none>  |
+##### Note
+As you can see in the sample read side config one can specify multiple read side in the yaml file using the `---` yaml
```

Comment:
Interesting. Why not use a YAML array instead?

Reply:
Making use of the YAML array I think will be great. However I don't think we should be backward compatible. This feature is easy to adapt AFAIK. This is going to be a breaking change in how to setup the read sides

Reply:
My goal is to ge rid of the env var which is error prone

Reply:
> Interesting. Why not use a YAML array instead?

The `---` is a yaml file document separator. So my goal is to have a single file containing all read side configs